api/src/main/java/marquez/db/RunDao.java

```diff
@@ -237,7 +237,14 @@ default void registerRunOutputDataset(UUID runUuid, JobMeta jobMeta) {
                       toSchemaFields(d.getFields()),
                       runUuid);
               datasetVersionDao.upsert(
-                  UUID.randomUUID(), Instant.now(), dsRow.get().getUuid(), version, runUuid);
+                  UUID.randomUUID(),
+                  Instant.now(),
+                  dsRow.get().getUuid(),
+                  version,
+                  runUuid,
+                  datasetVersionDao.toPgObjectFields(d.getFields()),
```

Comment:
This change won't propagate tags on fields. A change would be required to keep the current propagation behavior.

Reply:
The tags for the dataset fields wouldn't be propagated? Mind opening an issue outlining the change? Thanks!